fpdf/fpdf.py

```diff
@@ -1413,6 +1429,42 @@ def _draw_rounded_rect(self, x, y, w, h, style, round_corners, r):
             self.line(point_5[0], point_5[1], point_6[0], point_6[1])
             self.line(point_7[0], point_7[1], point_8[0], point_8[1])
 
+    def _draw_box(self, x1, y1, x2, y2, border, fill):
```

Comment:
Why not simply using `FPDF.rect()`?

Re-using this existing method would avoid some code duplication

Reply:
The border argument here is compatible with the output of get_cell_border.

As this function was only used once, I've moved it out of the fpdf class and into table.py